WORKSPACE

```diff
@@ -1,10 +1,81 @@
 workspace(name = "istio_ecosystem_wasm_extensions")
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
+# reference the same commit as istio release-1.7
```

Comment:
TODO: this needs to be turned into a bazel library (in istio/proxy for now I guess), so it is not duplicated.

Reply:
Added a TODO for this.